### DISCUSSION TITLE
How to delete duplicate or corresponding point from 2 point cloud files?

### DISCUSSION THREAD

**Discussion by kartikanrp** [isl-org/Open3D#6014] [Q&A] [answered]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### My Question

Hi everyone,

I have two point clouds (i.e., target and source point cloud). I want to delete the corresponding or duplicated point in a specific radius from the target point cloud (Target minus source). In the end, I want to keep the remaining point cloud after the corresponding point cloud has been removed. How to do this? 

I have tried to use this code (attached below) but it does not work (the filtered cloud looks the same with the previous one)

```
#This code can combine two point cloud and visualize it

# Load the two point clouds

cloud1 = o3d.io.read_point_cloud("cloud1.pcd")
cloud2 = o3d.io.read_point_cloud("cloud2.pcd")
o3d.visualization.draw_geometries([cloud1.pcd])
o3d.visualization.draw_geometries([cloud2.pcd])

combined_cloud = o3d.io.read_point_cloud([combined.pcd]")
o3d.visualization.draw_geometries([combined_cloud])

# Remove duplicate points from the combined cloud
filtered_cloud = combined_cloud.remove_duplicated_points()

o3d.visualization.draw_geometries([filtered_cloud])
```

If you have any suggestion, I would really appreciate it. Thank you.

**Comment by ssheorey** [comment]
How to questions are more suitable as a Discussion.

**Reply by xleEmesent** [reply, marked as answer]
You probably can try to compute the cloud to cloud distance first. Then discard any points having distance than some threshold such as less than 0.01.  Open3d has similar example for it. http://www.open3d.org/docs/latest/tutorial/geometry/pointcloud.html#Point-cloud-distance

**Reply by kartikanrp** [reply]
Thank you. I tried your suggestion and it is solved now.
